### ISSUE TITLE
[Bug]: Cannot access module settings or move module using menu

### ISSUE THREAD

**Issue by valadas** [dnnsoftware/Dnn.Platform#5740]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

In the current develop branch only (not on 9.12.0 release) the settings and the move menu above modules is missing

### Steps to reproduce?

1. Build from develop branch
2. Perform a clean install
3. Attempt to access module settings for a given model

### Current Behavior

Those menu items don't show.

### Expected Behavior

They should show.

### Relevant log output

```shell
jQuery.Deferred exception: $module.find(...).dnnHelperTip is not a function TypeError: $module.find(...).dnnHelperTip is not a function
    at $.fn.dnnModuleDragDrop (http://dnndev.loc/resources/shared/scripts/dnn.dragDrop.js?cdv=44:115:46)
    at HTMLDocument.<anonymous> (http://dnndev.loc/SomePage:542:28)
    at e (http://dnndev.loc/Resources/libraries/jQuery/03_05_01/jquery.js?cdv=44:2:30005)
    at t (http://dnndev.loc/Resources/libraries/jQuery/03_05_01/jquery.js?cdv=44:2:30307) undefined
```

```
Uncaught TypeError: $module.find(...).dnnHelperTip is not a function
    at $.fn.dnnModuleDragDrop (dnn.dragDrop.js?cdv=44:115:46)
    at HTMLDocument.<anonymous> (Agreements:542:28)
    at e (jquery.js?cdv=44:2:30005)
    at t (jquery.js?cdv=44:2:30307)
```

```
Uncaught TypeError: $parent.find(...).dnnConfirm is not a function
    at buildMenu (ModuleActions.js?cdv=44:224:70)
    at $.fn.dnnModuleActions (ModuleActions.js?cdv=44:379:21)
    at setUpActions (SomePage:304:57)
    at Agreements:414:13
    at dispatch (jquery.js?cdv=44:2:43090)
    at v.handle (jquery.js?cdv=44:2:41074)
```
```


### Anything else?

_No response_

### Affected Versions

10.0.0 (alpha)

### What browsers are you seeing the problem on?

Chrome

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by valadas** [comment]
Also can't drag a module into a pane (may or may not be related...

**Comment by mitchelsellers** [comment]
We are now seeing this on a 9.13.2 installation. 

@valadas any idea  how we might diagnose?

**Comment by valadas** [comment]
Any javascript errors? Does it happen with the default them?

**Comment by mitchelsellers** [comment]
Same JS errors as noted above.

Yes appears with default theme too

**Comment by valadas** [comment]
A bad upgrade maybe, do you have the newer DnnPlugins package upgraded, maybe try to reinstall just that package...

**Comment by mitchelsellers** [comment]
Should that show up as a JS LIbrary?

**Comment by mitchelsellers** [comment]
Found it!

No clue why that happened

**Comment by ajplopez** [comment]
I think I'm running into the same issue on a DNN upgrade. Is this something that can be fixed post-upgrade or do I have to revert and do something before upgrading?

**Comment by ajplopez** [comment]
![Screenshot 2024-07-12 at 6 34 57 PM](https://github.com/user-attachments/assets/fbf9a5a1-7403-499e-a681-758409399c2a)

When I try to install that library I get an error.

**Comment by valadas** [comment]
@ajplopez you should have some more info in the logs, either in the persona bar or in portals/_default/logs It is hard to go just by this information... Any antivirus getting in the way maybe ? Folder permissions ? etc.

**Comment by ajplopez** [comment]
Thanks @valadas . Turns out I needed to install hoverIntent jQuery Plugin **first**, and then DNN jQuery Plugin and it worked perfectly.

Per your and @david-poindexter suggestion I checked the Portals/_default/Logs and found this message: "[ERROR] DotNetNuke.Services.Installer.Log.Logger - A dependent package is not installed - HoverIntent"

So to anyone having this issue, try installing HoverIntent from the Available Extensions > JavaScript Libraries first. And then DNN jQuery Plugins and it should work.
